templates/api/keras_cv/metrics/metrics.md

```diff
@@ -0,0 +1,8 @@
+# KerasCV Metrics
```

Comment:
I think this file should be index.md not metrics.md?

Reply:
Good catch - updated.